test/test_dynamic_shapes.py

```diff
@@ -0,0 +1,20 @@
+from sklearn.datasets import make_blobs
+import torch, torch_xla
+import torch_xla.core.xla_model as xm
+import unittest
+
+dev = xm.xla_device()
+
+
+class TestDynamicShapes(unittest.TestCase):
+
+  def test_wrap(self):
+    a1 = torch.tensor([[1, 0, 0, 5, 0, 6]], device=dev)
+    a2 = torch.nonzero(a1)
+    a3 = a2.shape[0] + 3  # tests wrap
```

Comment:
Can you explain why `a2.shape[0]+3` tests wrap?

Reply:
what happens here is that a2.shape invokes XlaSymIntNodeImpl::wrap and that produces another SymInt(XlaSymIntNodeImpl(SizeConstant(2)) which we can that use freely in addition.

Reply:
I see. One question about how python's `a2.shape` maps to c++'s `XlaTensorImpl::sym_sizes`, you mentioned the magic happens at https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/python/init.cpp#L1409-L1413. That line reference (1409-1413) seem a bit off. Do you mean https://github.com/pytorch/pytorch/blob/e2a4dfa468330c0587849bea4896ff5fffb33010/torch/csrc/jit/python/init.cpp#L1367-L1370 ?